conda/base/context.py

```diff
@@ -869,6 +869,15 @@ def custom_multichannels(self):
             default_channels = list(self._default_channels)
 
         if self.restore_free_channel:
+            deprecated.topic(
+                "24.9",
+                "25.3",
+                topic="Adding the 'free' channel as it existed prior to conda 4.7.",
+                addendum="Add 'https://repo.anaconda.com/pkgs/free' to your channels "
+                "list manually after 'defaults', if you still need content from "
+                "the 'free' channel.",
+                deprecation_type=FutureWarning,
+            )
```

Comment:
Along with this deprecation marker, should we also add some `TODO` comments reminding us to remove (in conda in version 25.3) other mentions of the `restore_free_channel` in this file, e.g. https://github.com/conda/conda/blob/main/conda/base/context.py#L402, https://github.com/conda/conda/blob/main/conda/base/context.py#L1204 and https://github.com/conda/conda/blob/main/conda/base/context.py#L1737-L1742?

Reply:
I am not convinced that's needed since we'll do a pass over the feature ahead of 25.3. I also find the TODO comments suspect since they are super hand wavy.